src/index.html

```diff
@@ -3,11 +3,49 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, initial-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
     <h1>Product cards</h1>
+    <div class="product-card" data-qa="card">
+      <div class="product-card__img"></div>
+
+      <div class="product-card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
```

Comment:
Need to use the semantic tag for text, FIx it everywhere

Reply:
isn't fixed. You should use `h2` tag here and for other text elements you can use `p` tag